packages/component-utilities/src/buildQuery.js

```diff
@@ -1,12 +1,14 @@
 import { setContext, getContext } from 'svelte';
 import { Query, sql } from '@uwdata/mosaic-sql';
 import { QueryStore } from '@evidence-dev/query-store';
+import { query } from '@evidence-dev/universal-sql/client-duckdb';
 
 const QUERY_CONTEXT_KEY = '___usql_query';
-/** @type {(x: string) => QueryStore} */
-let queryFunction = () => {
-	throw new Error('Query Function has not yet been set. Use setQueryFunction first.');
-};
+/**
+ * Defaults to the query function from universal sql
+ * @type {(x: string) => QueryStore}
+ */
+let queryFunction = query;
```

Comment:
The usql query function itself only takes the sql string, so this just doesn't have any of the wrapping items

I haven't seen it be an issue in the app proper because it is correctly set there, but in storybook it was easier to do it this way

Reply:
If it's more for storybook would it make more sense to put a `setQueryFunction` in the storybook file that sets it to `import(@evidence-dev/universal-sql/client-duckdb).query` instead of doing that here? Shouldn't matter too much but I feel like there might be confusing behavior in previous error situations otherwise